src/main/java/com/mercateo/rest/jersey/utils/exception/DuplicateValidationException.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright © 2017 Mercateo AG (http://www.mercateo.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mercateo.rest.jersey.utils.exception;
+
+import javax.validation.ValidationException;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Getter
+@AllArgsConstructor
+public class DuplicateValidationException extends ValidationException {
```

Comment:
Wouldn't be DuplicateExeption be a better name?

Reply:
refactored